<ui version="4.0" >
 <class>ASysResMonPageClass</class>
 <widget class="QWidget" name="ASysResMonPageClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ASysResMonPage</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="upMonitorLabel" >
       <property name="text" >
        <string>Monitor:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="upConnectionsCheckBox" >
       <property name="text" >
        <string>Connections</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="upLogonsCheckBox" >
       <property name="text" >
        <string>Logons</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="upSessionsCheckBox" >
       <property name="text" >
        <string>Sessions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="upRequestsCheckBox" >
       <property name="text" >
        <string>Requests</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="upIntervalLabel" >
       <property name="text" >
        <string>Interval:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="upIntervalSpinBox" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>120</number>
       </property>
       <property name="value" >
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="upConnectionsGroupBox" >
     <property name="title" >
      <string>Connections</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QTreeWidget" name="upConnectionsTreeWidget" >
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <property name="columnCount" >
         <number>2</number>
        </property>
        <column>
         <property name="text" >
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>2</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QGroupBox" name="upLogonsGroupBox" >
     <property name="title" >
      <string>Logons</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QTreeWidget" name="upLogonsTreeWidget" >
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <property name="columnCount" >
         <number>2</number>
        </property>
        <column>
         <property name="text" >
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>2</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="upSessionsGroupBox" >
     <property name="title" >
      <string>Sessions</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QTreeWidget" name="upSessionsTreeWidget" >
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <property name="columnCount" >
         <number>2</number>
        </property>
        <column>
         <property name="text" >
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>2</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QGroupBox" name="upRequestsGroupBox" >
     <property name="title" >
      <string>Requests</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QTreeWidget" name="upRequestsTreeWidget" >
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <property name="columnCount" >
         <number>2</number>
        </property>
        <column>
         <property name="text" >
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>2</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
